src/Equinor.ProCoSys.IPO.Domain/AggregateModels/CertificateAggregate/Certificate.cs

```diff
@@ -29,7 +29,11 @@ protected Certificate()
             {
                 throw new ArgumentNullException(nameof(project));
             }
-          
+            if (project.Plant != plant)
+            {
+                throw new ArgumentException($"Plant {plant} should be equal to project plant {project.Plant}");
```

Comment:
throw new ArgumentException($"Can't relate {nameof(project)} in {project.Plant} to item in {plant}");